Backend/app/rider-platform/rider-app/Main/src/Tools/Notifications.hs

```diff
@@ -689,3 +694,8 @@ notifyRideStartToEmergencyContacts booking ride = do
       void $
         Sms.sendSMS booking.merchantId booking.merchantOperatingCityId (Sms.SendSMSReq message emergencyContact.mobileNumber sender)
           >>= Sms.checkSmsResult
+    checkTimeConstraintForFollowRide config now = do
```

Comment:
Looks like this function is repeated many times. Create a common function for it.